fighters/demon/src/acmd/smashes.rs

```diff
@@ -65,14 +67,14 @@ unsafe fn demon_attack_s4_game(fighter: &mut L2CAgentBase) {
 }
 
 #[acmd_script( agent = "demon", script = "game_attackhi4", category = ACMD_GAME, low_priority )]
-unsafe fn demon_attack_hi4_game(fighter: &mut L2CAgentBase) {
+unsafe fn game_attackhi4(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     smash::app::FighterSpecializer_Demon::set_devil(boma, true, 10.0);
     frame(lua_state, 1.0);
-    FT_MOTION_RATE(fighter, 0.7);
     if is_excute(fighter) {
         JostleModule::set_team(boma, 1);
+        FT_DESIRED_RATE(fighter, 14.0-1.0, 9.0);
```

Comment:
move this up two lines (all motion rate changes should go outside is_excute blocks)